app/src/main/java/id/rizmaulana/covid19/util/NumberUtils.kt

```diff
@@ -7,15 +7,23 @@ import java.util.*
 /**
  * rizmaulana 04/03/20.
  */
-object NumberUtils{
+object NumberUtils {
 
-    fun numberFormat(number: Int) = NumberFormat.getNumberInstance(Locale.getDefault()).format(number)
+    fun numberFormat(number: Int?) = number?.let {
+        NumberFormat.getNumberInstance(Locale.getDefault()).format(number)
+    } ?: "-"
 
     fun extractDigit(number: String) = Regex("[^0-9]").replace(number, "").toInt()
 
-    fun formatTime(time: Long) : String{
+    //TODO: Move to another class for date formatter
+    fun formatTime(time: Long): String {
         val sdf = SimpleDateFormat("dd MMMM yyyy", Locale.getDefault())
```

Comment:
We could use this lib https://github.com/JakeWharton/ThreeTenABP for easier date handling 🙏

Reply:
It is Java 8 Date Time API with backport for Android. So we can use `Localdate` on lower sdk version.

Ref about Java 8 Date Time API: https://www.journaldev.com/2800/java-8-date-localdate-localdatetime-instant